lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -72,6 +74,13 @@ protected function configure() : void
                 InputOption::VALUE_OPTIONAL,
                 'Max line length of unformatted lines.',
                 120
+            )
+            ->addOption(
+                'check-database-platform',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Check Database Platform to the generated code.',
```

Comment:
From https://github.com/doctrine/migrations/pull/782#discussion_r248850913:
```suggestion
                'Whether to add a database platform check at the beginning of the generated code.',
```